topics-project/app/settings/profile/actions.js

```diff
@@ -0,0 +1,20 @@
+import { createClient } from "../../../src/utils/supabase/server";
+
+export const fetchUserDetails = async () => {
```

Comment:
this function seems to be using hardcoded data as well as not handling the cases in the if statements, looks incomplete

Reply:
decided to not make the function in `profile/actions` and to create the function in `profile/page.jsx` instead.usj

i just forgot to delete the file , thx for pointing it out!